lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -305,9 +307,9 @@ public function formatDateTime(
         }
 
         $formatter = $this->dateFormatterFactory->create(
-            (string)($locale ?: $this->_localeResolver->getLocale()),
-            (int)($dateType ?? \IntlDateFormatter::SHORT),
-            (int)($timeType ?? \IntlDateFormatter::SHORT),
+            (string) ($locale ?: $this->_localeResolver->getLocale()),
+            (int) $dateType,
+            (int) $timeType,
```

Comment:
What will be if someone runs 
```
$timezone->formatDateTime($date, null, null)
``` 
?

Reply:
I've changed the implementation, please check